src/CodeSnippetWidget.js

```diff
@@ -39,18 +47,18 @@ export default class CodeSnippetWidget extends React.Component {
                 <li
                   role="presentation"
                   className={
-                    "tabs-component-tab" + (index == this.state.active ? " is-active" : "")
+                    "tabs-component-tab" + ((uniqueId + index) == this.state.active ? " is-active" : "")
                   }
                   key={index}
                 >
                   <a
-                    aria-controls={`${key}`}
+                    aria-controls={`${key + uniqueId}`}
```

Comment:
With the new unique HAR key, I think renaming the `key` variable to `snippetKey`would improve the readability. Then you have the `harKey` for unique across HARs (i.e., swagger endpoint sections) and `snippetKey` for unique across snippet tabs.